ui/src/shared/components/CheckPlot.tsx

```diff
@@ -128,7 +128,7 @@ const CheckPlot: FunctionComponent<Props> = ({
           <ThresholdMarkers
             key="custom"
             thresholds={thresholds}
-            onSetThresholds={updateCheckThresholds}
+            onSetThresholds={updateTimeMachineCheckThresholds}
```

Comment:
(For later) Seems like we need some way of making thresholds uneditable when a `CheckPlot` is used in a dashboard cell.

Reply:
It is currently technically uneditable, but making it explicit would be better